Directory.Build.targets

```diff
@@ -11,13 +12,4 @@
     <Compile Include="$(MSBuildThisFileDirectory)/shared/**/*.cs" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)'!='netstandard2.0'">
-    <!-- Exclude MSBuild runtime assets from both src and test projects
-         as they shouldn't be present in this solution's output paths.
-         Instead, these dependencies should be loaded from the selected
-         MSBuild's location. -->
-    <PackageReference Include="Microsoft.Build" ExcludeAssets="runtime" />
-    <PackageReference Include="Microsoft.Build.Framework" ExcludeAssets="runtime" />
```

Comment:
This is probably ok, but it would be worth double-checking that these libraries don't show up in our artifacts directories after the change. There was an issue previously where some projects were pulling these in transitively (likely fixed now) and it resulted in the binaries being present at runtime, which is not what we want. Microsoft.Build.Locator should be the only MSBuild library in the CLI's bin path.